src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -144,10 +150,24 @@ private Program(Options options)
             return branchName;
         }
 
+        private async Task CommentOnIssue(string user, string repo, int issueNumber, string message)
+        {
+                var resp = await _client.Connection.Post<string>(
```

Comment:
`_client` has an API for adding the comment directly; I'd rather see that being used than manually sending POST requests.